apps/nova/src/hooks/call.ts

```diff
@@ -357,3 +357,47 @@ export function useCallEvents(
     mateVideo,
   };
 }
+
+export function useFullScreen() {
+  const [isFullScreen, setIsFullScreen] = useState(false);
+
+  const startFullScreen = useCallback(async () => {
+    try {
+      setIsFullScreen(true);
+      await document.body.requestFullscreen();
```

Comment:
We don't want to whole body to be full screen. We want this [`div`](https://github.com/litespace-org/litespace/blob/26f7e8691d90effdd17d3d4b8210949cdb71e071/apps/nova/src/pages/Call.tsx#L189)  to be full screen. You can easily achieve this by using react refs. 

### Example

```ts

function useFullScreen() {
   const ref = useRef<HTMLDivElement>(null);

  const toggle = useCallback(() => {
    if (!ref.current) return;
    ref.current.requestFullScreen();
  }, [)


  return {ref, toggle, isFullScreen}
}


// in Call.tsx

const {ref, toggle} = useFullScreen()

return <div ref={ref}> </div>

```